lib/bugsnag/utility/circular_buffer.rb

```diff
@@ -2,30 +2,57 @@ module Bugsnag::Utility
   class CircularBuffer
     include Enumerable
 
+    # @return [Numeric] the current maximum allowable number of items
     attr_reader :max_items
 
+    ##
+    # Creates a circular buffer
```

Comment:
The main description portion of yard comments for `#initialize` is not needed as yard is smart enough to generate a generically useful one (but leave the `@param` comment in)